pkg/disk/disk.go

```diff
@@ -17,13 +17,29 @@ type Interface interface {
 	DiskStats(context.Context, *DiskStatsRequest) (*DiskStatsResponse, error)
 	GetAttachState(context.Context, *GetAttachStateRequest) (*GetAttachStateResponse, error)
 	GetDiskNumberByName(context.Context, *GetDiskNumberByNameRequest) (*GetDiskNumberByNameResponse, error)
+
+	// GetDiskState gets the offline/online state of a disk.
```

Comment:
I think this could be part of a different PR, I'd limit this change to iSCSI changes only

Reply:
Oh oops I meant to leave these changes for later. Accidentally pushed them.